notebook/static/base/js/keyboard.js

```diff
@@ -225,6 +225,17 @@ define([
         }
         return dct;
     };
+    
+    ShortcutManager.prototype.get_shortcut_for_action_name = function(name){
```

Comment:
the problem with get, is that you could get many things, action, action name, shortcut, it was get and i keept getting confuse for what I was getting for what.